File: 피보나치2.py
N = int(input())
array = [0 for i in range(91)];

for i in range(N+1):
  if i <= 1:
    array[i] = i
  else :
    array[i] = array[i-1] + array[i-2]
print(array[N])
# 5527939700884757

File: 피보나치2.js
const fs = require("fs");
let input = fs.readFileSync('./input.txt').toString();
let inputCount = parseInt(input);
let dp=[];
for(let i = 0; i<=inputCount; i++){
  if(i===0){
    dp[i] = 0;
    console.log(dp[i]);
  }else if(i === 1){
    dp[i] = 1;
    console.log(dp[i]);
  }else{
    dp[i] = dp[i-1]+ dp[i-2];
    console.log(dp[i]);
  }
}
console.log(dp[inputCount]);